spec/asyncapi.md

```diff
@@ -1322,6 +1372,7 @@ All objects defined within the components object will have no effect on the API
 
 Field Name | Type | Description
 ---|:---|---
+<a name="componentsServers"></a> servers | Map[`string`, [Server Object](#serverObject) \| [Reference Object](#referenceObject)] | An object to hold reusable [Server Objects](#serverObject).
```

Comment:
Do we really need this? Why can't we point directly to `#/servers/{name-of-server}`?

Reply:
the only other instances i found in the spec where a field points to another field where when they pointed to component objects: this is the only reason why i've modelled it this way. If you say we can point directly to `#/servers/{name-of-server}` and get the same kind of static validation of the reference then i'm more than happy to make that change. Let me know @fmvilas .